tqdm/_tqdm.py

```diff
@@ -342,7 +353,7 @@ def format_meter(n, total, elapsed, ncols=None, prefix='',
                 bar_length, frac_bar_length = divmod(
                     int(frac * N_BARS * 10), 10)
 
-                bar = '#' * bar_length
+                bar = '█' * bar_length
```

Comment:
@MPagel I opened the PR based on the file @blaz62746 uploaded in an issue. It's good that we have a PR though because it highlights how people may get confused about `tqdm` usage. But yet, @blaz62746 if you absolutely want to implement these non-standard changes into your own version of `tqdm` then you should fork this repo and make your changes in your fork. Then you can easily merge in new changes from our repo when needed - and if you ever feel the need open a PR.